presto-hive/src/test/java/com/facebook/presto/hive/TestHudiDirectoryLister.java

```diff
@@ -107,6 +112,71 @@ private Table getMockTable()
                 Optional.empty());
     }
 
+    private Table getMockMORTableWithPartition()
+    {
+        return new Table(
+                "schema",
+                "hudi_mor_part_update",
+                "user",
+                EXTERNAL_TABLE,
+                new Storage(fromHiveStorageFormat(PARQUET),
+                        getTableBasePath("hudi_mor_part_update"),
+                        Optional.of(new HiveBucketProperty(
+                                ImmutableList.of(),
+                                1,
+                                ImmutableList.of(),
+                                HIVE_COMPATIBLE,
+                                Optional.empty())),
+                        false,
+                        ImmutableMap.of(),
+                        ImmutableMap.of()),
+                ImmutableList.of(),
+                ImmutableList.of(),
+                ImmutableMap.of(),
+                Optional.empty(),
+                Optional.empty());
+    }
+
+    @Test
+    public void testDirectoryListerForMORTableWithPartitionUpdates()
```

Comment:
Can we also add a negative test to verify when the merged view is disabled/doesn't include the table name?

Reply:
added a negative test `testDirectoryListerForMORTableWithoutTableNames`